projects/i18n/src/lib/loader/translation-loader.ts

```diff
@@ -0,0 +1,22 @@
+/*!
+ * Copyright 2020 VMware, Inc.
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+export class TranslationLoader {
```

Comment:
Every exported object should have some docs, even if a single sentence. That also applies to its public members/methods